js/templates/modals/settings/advanced/server.html

```diff
@@ -0,0 +1,45 @@
+<form class="box padKids padStack clrP clrBr js-server">
+  <div class="flexRow">
+    <h3><%= ob.polyT('settings.server') %></h3>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label for="currentFirewallType">
+        <%= ob.polyT('settings.currentFirewallType') %>
+      </label>
+    </div>
+    <div class="col9">
+      <input class="clrBr" readonly
+        type="text"
+        name="currentFirewallType" 
+        id="currentFirewallType" 
+        value="<%= ob.currentFirewallType %>">
+    </div>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label>
+        <%= ob.polyT('settings.connections') %>
+      </label>
+    </div>
+    <div class="col9">
+      <div class="btnStrip flex clrP clrBr">
+        <a class="btn clrP clrBr js-showConnectionManagement flexNoShrink"><%= ob.polyT('settings.btnShowConnectionManagement') %></a>
+        <a class="btn clrP clrBr js-showConnectedPeers flexNoShrink"><%= ob.polyT('settings.btnShowConnectedPeers') %></a>
+      </div>
+    </div>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label>
+        <%= ob.polyT('settings.onOff') %>
+      </label>
+    </div>
+    <div class="col9">
+      <div class="btnStrip flex clrP clrBr">
```

Comment:
Actually, Mike is kind of right. If the local server dies and you are connect to it, the Connection modal will pop up, so you won't be able to get to settings.

The only time you would be able to get to settings when the local server is down is if you are connected to a remote server... but, in that case, why would you want to start the local server? If you want to connect to the local server, you would just use the connection modal (or page nav connection menu) and connect to the default server... which would automatically start the local server.

I really think the Turn Server On button is not necessary.